### ISSUE TITLE
rawDatapoints are no longer reported in query summary

### ISSUE THREAD

**Issue by bobrik** [OpenTSDB/opentsdb#784]
Before (v2.2.0RC3):

```json
  {
    "statsSummary": {
      "datapoints": 192,
      "rawDatapoints": 6176,
      "aggregationTime": 3,
      "serializationTime": 5,
      "storageTime": 5719,
      "timeTotal": 5697
    }
  }
```

Currently (v2.2.0):

```json
  {
    "statsSummary": {
      "avgAggregationTime": 1.088369,
      "avgHBaseTime": 295.162437,
      "avgQueryScanTime": 5904.48002,
      "avgScannerTime": 295.223705,
      "avgScannerUidToStringTime": 0,
      "avgSerializationTime": 1.106604,
      "emittedDPs": 182,
      "maxAggregationTime": 1.088369,
      "maxHBaseTime": 5903.248743,
      "maxQueryScanTime": 5904.48002,
      "maxScannerUidtoStringTime": 0,
      "maxSerializationTime": 1.106604,
      "maxUidToStringTime": 0.048007,
      "processingPreWriteTime": 5905.98901,
      "queryIdx_00": {
        "aggregationTime": 1.088369,
        "avgHBaseTime": 5903.248743,
        "avgScannerTime": 5904.474104,
        "avgScannerUidToStringTime": 0,
        "emittedDPs": 182,
        "groupByTime": 0,
        "maxHBaseTime": 5903.248743,
        "maxScannerUidtoStringTime": 0,
        "queryIndex": 0,
        "queryScanTime": 5904.48002,
        "scannerStats": {
          "scannerIdx_00": {
            "compactionTime": 0.661153,
            "hbaseTime": 5903.248743,
            "scannerId": null,
            "scannerTime": 5904.474104,
            "scannerUidToStringTime": 0,
            "successfulScan": 1,
            "uidPairsResolved": 0
          }
        },
        "serializationTime": 1.106604,
        "successfulScan": 1,
        "uidPairsResolved": 0,
        "uidToStringTime": 0.048007
      },
      "successfulScan": 1,
      "uidPairsResolved": 0
    }
  }
```

**Comment by bobrik** [comment]
Having the number of distinct timeseries would be nice too.

**Comment by bobrik** [comment]
Probably deserves another issue, but overall avg doesn't look right:

* `"avgHBaseTime": 295.162437`
* `"maxHBaseTime": 5903.248743`
* `"queryIdx_00"`
  * `"avgHBaseTime": 5903.248743`
  * `"maxHBaseTime": 5903.248743`

**Comment by manolama** [comment]
I'll restore the raw data points here. It's an estimate now as it's pretty expensive to calculate (as I had removed it before because it doubled query time). I'll also look at the avg, that does seem fishy.

**Comment by manolama** [comment]
Merged in 4b14b2d3a2a3b67571912feb7f773c9356eda58e. Give it a shot and let us know how it looks.
